ring_api/wrappers/dring_cython.pyx

```diff
@@ -175,6 +177,51 @@ cdef class ConfigurationManager:
         confman_cpp.sendAccountTextMessage(
                 raw_account_id, raw_ring_id, raw_content)
 
+    def get_tls_default_settings(self):
+        """Get the TLS default settings
```

Comment:
In present the verbs in English have an **s** at the end like here he/she/it gets something.